Assets/StreamingAssets/LUA/Furniture.lua

```diff
@@ -579,7 +579,18 @@ end
 -- Dummy heater uninstall function
 -- THis function gets called once, when the funriture is unisntalled
 function Heater_UpdateTemperature( furniture, deltaTime)
-	World.current.temperature.SetTemperature(furniture.tile.X, furniture.tile.Y, 300)
+    --if (furniture.HasPower() == false) then
+    --    return
+    --end
+    
+    tile = furniture.tile
+    pressure = tile.Room.GetGasPressure() / tile.Room.GetSize()
+    efficiency = ModUtils.Clamp01(pressure / furniture.Parameters["pressure_threshold"].ToFloat())
+    temperatureChangePerSecond = furniture.Parameters["base_heating"].ToFloat() * efficiency
+    temperatureChange = temperatureChangePerSecond * deltaTime
```

Comment:
Try to use 'local' if the var is not needed out of it's context.
The Lua 'local' is the equivalent to C# 'private#.